brave-http/src/main/java/com/github/kristofa/brave/http/HttpServerResponseAdapter.java

```diff
@@ -18,12 +18,8 @@ public HttpServerResponseAdapter(HttpResponse response)
 
     @Override
     public Collection<KeyValueAnnotation> responseAnnotations() {
-        int httpStatus = response.getHttpStatusCode();
-
-        if ((httpStatus < 200) || (httpStatus > 299)) {
-            KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create("http.responsecode", String.valueOf(httpStatus));
-            return Arrays.asList(statusAnnotation);
-        }
-        return Collections.EMPTY_LIST;
+        KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create(
+                "http.responsecode", String.valueOf(response.getHttpStatusCode()));
```

Comment:
Same argument as above - we reused Brave's convention (https://github.com/openzipkin/brave/blob/master/brave-http/src/main/java/com/github/kristofa/brave/http/HttpClientResponseAdapter.java#L25) but it should probably be fixed everywhere in Brave.